components/src/core/AppBar/AppBar.tsx

```diff
@@ -16,6 +16,7 @@ export type AppBarClasses = {
     root?: string;
     background?: string;
     expanded?: string;
+    collapsed?: string;
```

Comment:
What is the difference between `collapsed` and `root`?

Reply:
I added in the `collapsed` class because I only want these styles to be applied to a collapsed appbar.  If I put some of these styles on the root, I'd have to add additional styles to the `expanded` class set to disable the root styles that I no longer want.

Reply:
You either have to override default styles from root, or add additional styles to collapsed. I think it's the same amount of work either way, but I agree with adding the collapsed class since it's much more obvious how you'd go about using it.